people.html

```diff
@@ -192,39 +192,40 @@ <h1>Rose Johnson</h1>
                             </article>
                         </a>
                     </div>
+
 		    <div class="item-5">
                         <a class="card">
-                            <div class="personPic" style="background-image: url('images/JustinLau.png')"></div>
+                            <div class="personPic" style="background-image: url('images/AndreasSoteriou.png')"></div>
                             <article>
-                                <h1>Justin Lau</h1>
-                                <span>Undergrad Researcher</span>
+                                <h1>Andreas Soteriou</h1>
+                                <span>Undergraduate Researcher</span>
                             </article>
                         </a>
                     </div>
 		    <div class="item-6">
                         <a class="card">
-                            <div class="personPic" style="background-image: url('images/HenriAbensour.png')"></div>
+                            <div class="personPic" style="background-image: url('images/MichelleWatson.png')"></div>
                             <article>
-                                <h1>Henri Abensour</h1>
-                                <span>Undergrad Researcher</span>
+                                <h1>Michelle Watson</h1>
+                                <span>Undergraduate Researcher</span>
                             </article>
                         </a>
                     </div>
 		    <div class="item-7">
                         <a class="card">
-                            <div class="personPic" style="background-image: url('images/AnantGupta.png')"></div>
+                            <div class="personPic" style="background-image: url('images/JustinLau.png')"></div>
                             <article>
-                                <h1>Anant Gupta</h1>
-                                <span>Undergrad Researcher</span>
+                                <h1>Justin Lau</h1>
+                                <span>Undergraduate Researcher</span>

```

Comment:
Sorry for writing "Undergraduate" in my instructions in the HPQC Discord server. We can stick with "Undergrad" which is what we had before. You can make that change, and then just commit that one change, and push it to the same fork of this repository, and the pull request should update automatically.